app/modules/import-export/usage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Elementor\App\Modules\ImportExport;
+
+use Elementor\App\Modules\ImportExport\Processes\Revert;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Usage {
+
+	/**
+	 * Register hooks.
+	 *
+	 * @return void
+	 */
+	public function set_usage_data() {
+		add_filter( 'elementor/tracker/send_tracking_data_params', function ( array $params ) {
+			$params['usages']['import_export']['revert'] = $this->get_revert_usage_data();
```

Comment:
maybe make sure that `$params['usages']['import_export']` is an array before pushing into it (if maybe for some reason it'll run before the "main" IE usage data or something)

Reply:
We have talked about it, it will create all the array cells if needed :)